docs/cloud-native-security/kspm.asciidoc

```diff
@@ -1,6 +1,81 @@
+
+This integration supports Amazon EKS and unmanaged Kubernetes clusters. For setup instructions, refer to <<get-started-with-kspm,Get started with KSPM>>.
+
+[discrete]
+[[kspm-how-kspm-works]]
+== How KSPM works
+. When you add a KSPM integration, it generates a Kubernetes manifest that, when applied to a cluster, deploys an {agent} as a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset[DaemonSet] — this ensures that all nodes are evaluated.
+. Upon deployment, the integration immediately assesses the security posture of your Kubernetes resources. The evaluation process repeats every four hours.
+. After each evaluation, the integration sends findings to {es}. Findings appear on the <<cloud-nat-sec-posture-dashboard,Cloud posture dashboard>> and the <<findings-page,findings>> page.
+
+[discrete]
+[[kspm-use-cases]]
+== Use cases
+
+The KSPM integration helps you to:
+
+. Identify and remediate `failed` findings
+. Identify the most misconfigured resources
+. Identify risks in particular CIS benchmark sections
+
+[discrete]
+[[kspm-remediate-failed-findings]]
+=== Identify and remediate failed findings
+
+To identify and remediate failed failed findings:
+
+. Go to the <<cloud-nat-sec-posture-dashboard,Cloud posture dashboard>>.
+. Click *View all failed findings*, either for an individual cluster or for all monitored clusters.
+. Click a failed finding. The findings flyout opens.
+. Follow the steps under *Remediation* to correct the misconfiguration.
+
+NOTE: Remediation steps typically include commands for you to execute. These sometimes contain placeholder values that you must replace before execution.
+
+[discrete]
+[[kspm-identify-misconfigured-resources]]
+=== Identify the most misconfigured Kubernetes resources
+
+To identify the Kubernetes resources generating the most failed findings:
+
+. Go to the <<findings-page,Findings>> page.
+. Click the *Group by* menu near the search box and select *Resource* to view a list of resources sorted by their total number of failed findings.
+. Click a resource ID to view the findings associated with that resource.
+
+[discrete]
+[[kspm-identify-config-risks-by-section]]
+=== Identify configuration risks by CIS section
+
+To identify risks in particular CIS sections:
+
+. Go to the <<cloud-nat-sec-posture-dashboard,Cloud posture dashboard>>.
+. In the Failed findings by CIS section widget, click the name of a CIS section to view all failed findings for that section.
+
+Alternatively:
+
+. Go to the Findings page.
+. Filter by the `rule.section` field.
+. For example, search for `rule.section : API Server` to view findings for benchmark rules in the API Server category.
+
+[discrete]
+[[kspm-faq]]
+== FAQ
+
+*Can I evaluate the security posture of my Amazon EKS clusters?*
+Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters. Support for other managed Kubernetes services, like GKE and AKS, is planned.
```

Comment:
We try to avoid forward-looking statements or anything that sounds like we're promising something, so the second sentence could be tricky.

Reply:
I know that's our standard practice, and I'm open to removing this. On the other hand, I am open to experimenting with including it since it was requested directly by @tinnytintin10 (the PM for this feature, so we don't have to worry about accuracy). @jmikell821 what do you think?